exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/AggPrelBase.java

```diff
@@ -51,7 +51,7 @@
 
 public abstract class AggPrelBase extends DrillAggregateRelBase implements Prel {
 
-  protected static enum OperatorPhase {PHASE_1of1, PHASE_1of2, PHASE_2of2};
+  public static enum OperatorPhase {PHASE_1of1, PHASE_1of2, PHASE_2of2};
```

Comment:
Nit, but enum values are constants and so convention is all upper case: maybe: PHASE1OF2 or PHASE1_OF_2 or PHASE_1_OF_2...

Not a big deal; the current form is a bit easier to read...